### DISCUSSION TITLE
Is there a way to create subgroup?

### DISCUSSION THREAD

**Discussion by mgomse** [profclems/glab#1003] [Q&A]
Hello,

I tried to do `glab repo create devops/helm-charts/my-helm-chart` and I got info:

> could not find group or namespace helm-charts: GET https://DOMAIN/api/v4/groups/helm-charts: 404 {message: 404 Group Not Found}

And that's true - I don't have a subgroup helm-charts in the devops group. Is there any possibility to use gitlab to:
- create subgroup
- or even better - use `glab repo create` with some flag which will create all subgroups if set to true

I wanted to start a discussion instead of writing an issue because I'm quite new in the `glab` community and just wanted to know - maybe I'm missing sth?

EDIT: Even when I created a subgroup manually with UI I still have the same message

**Comment by zemzale** [comment]
https://github.com/profclems/glab/issues/983
